web/styles/zulip.css

```diff
@@ -2198,13 +2199,14 @@ body:not(.spectator-view) {
 
     .stream-privacy-type-icon {
         font-size: 0.93em;
+        /* We set only the width so that flexbox
+           can do its work to properly center,
+           regardless of the height. */
         width: 0.93em;
-        height: 0.93em;
         padding-right: 5px;
-    }
-
-    .zulip-icon {
-        margin-top: 2px;
+        /* Override the [data-tippy-root]
+           style in `tooltips.css`. */
+        top: 0;
```

Comment:
I can merge, but I'm kinda guessing that we'll find that `data-tippy-root` logic needs to change too.
```
    .stream-privacy-type-icon {                                                                         
        width: 9px;                                                                                     
        padding-left: 2px;                                                                              
        top: 0.5px;                                                                                     
    }                                                                                                   
```

I think `web/templates/stream_settings/stream_settings_cannot_subscribe_tooltip.hbs` is where it's used?